capstone/src/main/java/com/google/univiz/scorecard/ScorecardConverter.java

```diff
@@ -0,0 +1,45 @@
+package com.google.univiz.scorecard;
+
+import com.google.common.base.Converter;
+
+/**
+ * ScorecardConverter class returns a representation of ScorecardData as an instance of type
+ * CollegeData.
+ */
+class ScorecardConverter extends Converter<ScorecardData, CollegeData> {
+
+  @Override
+  protected CollegeData doForward(ScorecardData scorecardCollege) {
+    CollegeData.Builder collegeBuilder = CollegeData.builder();
+
+    Integer scorecardCarnegieSize = scorecardCollege.carnegieSizeDegree();
+    if (scorecardCarnegieSize < 1 || scorecardCarnegieSize == null) {
+      CarnegieSizeDegree collegeDataCarnegieSize = CarnegieSizeDegree.getDegree(0);
```

Comment:
Since `collegeDataCarnegieSize` is local to this branch, it basically has no effect. Did you mean to update `collegeBuilder` here, instead? (tests would have caught this :-) )